R/plotNhoods.R

```diff
@@ -741,3 +741,105 @@ plotNhoodMA <- function(da.res, alpha=0.05, null.mean=0){
 
   return(ma.p)
 }
+
+
+#' Plot the number of cells in a neighbourhood per sample and condition
+#'
+#' @param x A \code{\linkS4class{Milo}} object with a non-empty \code{nhoodCounts}
+#' slot.
+#' @param nhoods A string vector specifying the IDs of the neighbourhoods to plot.
```

Comment:
Strictly this can also be a logical or numeric vector as these are valid for subsetting a sparse matrix, this functionality should be included, with a check for the input type (hint: great place to have a unit test).